Makefile

```diff
@@ -109,6 +109,12 @@ clean:
 	$(RM) $(gen_authopt_path)
 	$(RM) $(gen_zitadel_path)
 	$(RM) -r tmp/
+	find docs/apis/assets -type f ! -name 'assets.md' -delete 
+	find openapi/ -type f ! -name 'handler.go' -delete 
+	find . -name 'statik.go' -delete
+	$(RM) -r console/dist
+	find internal/api/ui/console/static -type f ! -name 'gitkeep' -delete
```

Comment:
BTW, I think I would have used `git clean -fdx openapi` etc. instead of find, because I think it's simpler and more forward compatible.

I leave it up to you, you can also just resolve this comment.